docs/book/security.md

```diff
@@ -276,3 +296,127 @@ identity_rights[right] {            # Right is in the identity_rights set if...
     right = rights[role]            # Role has rights defined.
 }
 ```
+
+### TLS-based Authentication Example
+
+To set up authentication based on TLS, we will need three certificates:
+
+1. the CA cert (self-signed),
+2. the server cert (signed by the CA), and
+3. the client cert (signed by the CA).
+
+These are example invocations using `openssl`.
+Don't use these in production, the key sizes are only good for demonstration purposes.
+
+Note that we're creating an extra client, which has a certificate signed by the proper
+CA, but will later be used to illustrate the authorization policy.
+
+```bash
+# CA
+openssl genrsa -out ca-key.pem 2048
+openssl req -x509 -new -nodes -key ca-key.pem -days 1000 -out ca.pem -subj "/CN=my-ca"
+
+# client 1
+openssl genrsa -out client-key.pem 2048
+openssl req -new -key client-key.pem -out csr.pem -subj "/CN=my-client"
+openssl x509 -req -in csr.pem -CA ca.pem -CAkey ca-key.pem -CAcreateserial -out client-cert.pem -days 1000
+
+# client 2
+openssl genrsa -out client-key-2.pem 2048
+openssl req -new -key client-key-2.pem -out csr.pem -subj "/CN=my-client-2"
+openssl x509 -req -in csr.pem -CA ca.pem -CAkey ca-key-2.pem -CAcreateserial -out client-cert-2.pem -days 1000
```

Comment:
`ca-key-2.pem` doesn't exist at this point. I think this should be the same CA as the first.

Reply:
Indeed! Thanks. Fixed. 😃